packages/blockchain-ui/src/components/elements/ButtonList/ButtonList.tsx

```diff
@@ -1,33 +1,53 @@
 import React, { Component } from 'react';
 import './ButtonList.scss';
 import { Button } from 'carbon-components-react';
+import IPackageRegistryEntry from '../../../interfaces/IPackageRegistryEntry';
 
-interface IProps {
-    buttons: any[];
+interface ButtonListProps {
+    onProgressChange: any;
```

Comment:
You can give your callback props types, for this one I think it'd be: 
`onProgressChange: (indexValue: number) => void;`